sched/adaptdl_sched/controller.py

```diff
@@ -391,6 +393,10 @@ async def _create_pod(self, job_metadata, pod_template,
                 "name": "ADAPTDL_SUPERVISOR_URL",
                 "value": config.get_supervisor_url(),
             })
+            container["env"].append({
+                "name": "ADAPTDL_VERSION",
+                "value": config.get_adaptdl_version(),
+            })
```

Comment:
`ADAPTDL_SCHED_VERSION`.

Also, if you're passing the version as an env var then you shouldn't need the changes to the supervisor. You can simply read the env var on the trainer-side.